keyboards/tr60w/readme.md

```diff
@@ -0,0 +1,26 @@
+# TR60W
+
+[![TR60W Imgur Link](https://imgur.com/a/vezmI99/)](https://imgur.com/a/vezmI99/)
```

Comment:
This needs to be a direct image link in order for GitHub to render the image. Also the formatting is a bit wrong.

```suggestion
![TR60W](https://i.imgur.com/Ez2T2Te.jpg)
```

Reply:
done.